server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@swapnilghonge you shouldn't need to duplicate this file, you can share with the kernel driver implementation.